app/agent/manus.py

```
@@ -7,6 +7,8 @@
 from app.tool.file_saver import FileSaver
 from app.tool.google_search import GoogleSearch
 from app.tool.python_execute import PythonExecute
+# Import HoneyHive tracing if needed for custom methods
+from app.honeyhive_tracer import pydantic_compatible_atrace
```

Ruff (0.8.2)

`app.honeyhive_tracer.pydantic_compatible_atrace` imported but unused

Remove unused import: `app.honeyhive_tracer.pydantic_compatible_atrace`

(F401)